packages/logger/src/index.ts

```diff
@@ -101,7 +101,7 @@ class ConsoleLogger implements Logger {
 
   logException(
     message: string,
-    exception: Error | string,
+    exception: Error | string | unknown,
```

Comment:
I wonder if this should be 
`exception: unknown,`
instead, because if it is unknown then it includes Error and string